src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+}
+
+.calendar__day {
+  display: flex;
+  position: relative;
+  height: $day-size;
+  width: $day-size;
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  border: solid $border-size black;
+  justify-content: center;
+  align-items: center;
+  transition:
+    background-color $animation-duration,
+    transform $animation-duration;
+
+  &:hover {
+    cursor: pointer;
+    transform: translateY(-20px);
+    background-color: $hover-color;
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
```

Comment:
It's great that you're using the SCSS `@for` directive to generate the day numbers. However, the current implementation inserts the day number as a pseudo-element. This could potentially cause issues with screen readers, as they might not read out the content of pseudo-elements. It would be better to insert the day numbers directly into the HTML.

Reply:
it is against the condition